src/App.svelte

```diff
@@ -83,33 +75,47 @@
       <Loading center />
     </div>
   {:then wallet}
-    <ColorPalette />
     <Header session={$session} {wallet} />
     <div class="wrapper">
       {#if $activeRouteStore.resource === "home"}
-        <Home />
+        <DynamicComponentImport path="../base/home/Index.svelte" />
```

Comment:
We lose all type safety here. I think I prefer to keep the type safety over the performance gain.